great_expectations/expectations/core/expect_column_distinct_values_to_be_in_set.py

```diff
@@ -330,10 +331,25 @@ def _descriptive_value_counts_bar_chart_renderer(
 
         return new_block
 
+    @public_api
     def validate_configuration(
         self, configuration: Optional[ExpectationConfiguration] = None
     ) -> None:
-        """Validating that user has inputted a value set and that configuration has been initialized"""
+        r"""Validates configuration for the Expectation.
```

Comment:
Assuming this isn't supposed to be an `r` string anymore since there are no backslashes or anything that I can see requiring it.